src/main/io/osd.c

```diff
@@ -227,16 +227,64 @@ static uint8_t osdGetDirectionSymbolFromHeading(int heading)
     return SYM_ARROW_SOUTH + heading;
 }
 
+// this will convert from relative positioning (i.e. measured from center pos)
+// to absolute positioning (based on display height and width)
+static void osdConvertToAbsolutePosition(uint8_t item, int8_t *pos_x, int8_t *pos_y) {
+    // output display dimensions
+    uint8_t display_width  = osdDisplayPort->colCount - 1;
```

Comment:
I moved the -1 from all calculations up to the variable definition. The name _width makes no longer sense, you are right. should be renamed.